### PR TITLE
Chaplain religion update

### PR DESCRIPTION
**Pull request by CptWad**
Adds new shit for chaplain deity choice

1. Firstly, re-instated the old bible/booze object from eons ago. Now when selecting certain (less than morally binding) religions you start with the old bible filled with booze and money.
2. Updated and fixed a few old religions.. IE: Norse now has more ways to trigger getting it (typing vikings at start will get you norse gods) and has a wider array of gods that it will choose.
3. Added three new religions. Roman, Clown and Mime.
4. Added items that will now spawn with certain religions (syndicate religion now gets the full fake space suit and malf religion gets full cardborg) so on and so forth.

### CODE REVIEW

File: code/game/jobs/job/civilian_chaplain.dm [hunk 1]
```diff
@@ -239,40 +274,65 @@
 				//Give them basic medical garb
 				H.equip_or_collect(new /obj/item/clothing/head/surgery/blue(H), slot_head)
 				H.equip_or_collect(new /obj/item/clothing/mask/surgical(H), slot_wear_mask)
+				H.equip_or_collect(new /obj/item/clothing/suit/storage/labcoat(H), slot_suit)
 				deity_name = "The Chief Medical Officer"
 			if("busta", "bustatime", "zas", "airflow", "hardcore", "hardcores")
```
Comment (clusterfack) [inline code/game/jobs/job/civilian_chaplain.dm]: Wooow ZAS doesn't equip some magboots on you.

Comment (CptWad) [inline code/game/jobs/job/civilian_chaplain.dm]: I could easily do that

**Comment by CptWad** [comment]
woops well guess slot_suit doesnt exist

**Comment by CptWad** [comment]
ok fixed it
also some guy from discord pm'd me some shit I could add so theres a bit more flavor
also added magboots to bustatime religion as per cluster

**Comment by ComicIronic** [comment]
Powergame religion confirmed.

**Comment by Durgen** [comment]
can you add a vox religion, 
Diety: Cawmother?

**Comment by Durgen** [comment]
nevermind, just looked at the code. Vox religion is already there

File: code/game/jobs/job/civilian_chaplain.dm [hunk 2]
```diff
@@ -303,7 +367,7 @@
 			if("revolution", "rev", "revolt")
 				B.name = "Down With Nanotrasen"
 				J = "Revolutionary"
-				deity_name = "Maximilien Robespierre" //From THE Revolution that defined the rest
+				deity_name = pick("Maximilien Robespierre", "Saul Alinsky")
```
Comment (Kurfursten) [inline code/game/jobs/job/civilian_chaplain.dm:370]: Wouldn't the result of this be that the chaplain wouldn't know his own deity's name without asking?

Comment (CptWad) [inline code/game/jobs/job/civilian_chaplain.dm]: you can slam people with the bible to find out

**Pull request merged**